### DISCUSSION TITLE
How I set overscroll-x: hidden for all my body tag?

### DISCUSSION THREAD

**Discussion by Minozzzi** [idiotWu/smooth-scrollbar#523] [Q&A] [answered]
I'm trying to set `overscroll-x: hidden` in my body via css but it totally breaks y and x axis scrolling

**Comment by sadeghbarati** [comment, marked as answer]
Hi check this comment

https://github.com/idiotWu/smooth-scrollbar/discussions/357#discussioncomment-854036

**Reply by Minozzzi** [reply]
Good!
